src/components/Reports/HealthIndicatorReport/HealthIndicatorReport.tsx

```diff
@@ -0,0 +1,91 @@
+import { useTranslation } from 'react-i18next';
+import MonthlyGoal from 'src/components/Dashboard/MonthlyGoal/MonthlyGoal';
+import {
+  HeaderTypeEnum,
+  MultiPageHeader,
+} from 'src/components/Shared/MultiPageLayout/MultiPageHeader';
+import { HealthIndicatorFormula } from './HealthIndicatorFormula/HealthIndicatorFormula';
+import { HealthIndicatorGraph } from './HealthIndicatorGraph/HealthIndicatorGraph';
+import { useMonthlyGoalQuery } from './MonthlyGoal.generated';
+
+const GraphTitle = styled(Typography)(({ theme }) => ({
+  marginBottom: theme.spacing(2),
+}));
+interface HealthIndicatorReportProps {
+  accountListId: string;
+  isNavListOpen: boolean;
+  onNavListToggle: () => void;
+  title: string;
+}
+
+export const HealthIndicatorReport: React.FC<HealthIndicatorReportProps> = ({
+  accountListId,
+  isNavListOpen,
+  onNavListToggle,
+  title,
+}) => {
+  const { t } = useTranslation();
+  const [noHealthIndicatorData, setNoHealthIndicatorData] = useState(false);
+  const { data, loading } = useMonthlyGoalQuery({
+    variables: {
+      accountListId,
+    },
+  });
+  return (
+    <Box>
+      <MultiPageHeader
+        isNavListOpen={isNavListOpen}
+        onNavListToggle={onNavListToggle}
+        title={title}
+        headerType={HeaderTypeEnum.Report}
+      />
+      <Container>
+        {noHealthIndicatorData ? (
+          <Grid container spacing={3}>
+            <Grid item xs={12}>
+              <Typography variant="h5" mt={2}>
+                {t('No Health Indicator data available')}
+              </Typography>
+              <Typography variant="body1" mt={2}>
+                {t(
+                  'There is currently no Health Indicator data available for this account. Please switch to an account that is an MPD Global account. If you are unsure whether you have access to an MPD Global account or need assistance in switching accounts, please reach out to your coach or our support team for guidance.',
+                )}
+              </Typography>
+            </Grid>
+          </Grid>
+        ) : (
+          <Grid container spacing={3}>
+            <Grid item xs={12}>
+              <MonthlyGoal
+                accountListId={accountListId}
+                loading={loading}
+                goal={data?.accountList.monthlyGoal ?? undefined}
+                received={data?.accountList.receivedPledges}
+                pledged={data?.accountList.totalPledges}
+                totalGiftsNotStarted={data?.contacts.totalCount}
+                currencyCode={data?.accountList.currency}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <GraphTitle variant="h5">{t('Health Indicator')}</GraphTitle>
+              <HealthIndicatorGraph accountListId={accountListId} />
+            </Grid>
+
+            <Grid item xs={12}>
+              <GraphTitle variant="h5">{t('MPD Health Formula')}</GraphTitle>
+
+              <HealthIndicatorFormula
```

Comment:
Can we add some padding at the bottom of the formula component? When you scroll all the way, it sits right at the bottom of the browser tab.

<img width="1210" alt="Screenshot 2024-12-19 at 1 03 32 PM" src="https://github.com/user-attachments/assets/d0f0b55c-b0ed-4b2c-8b06-ece8da2baeb7" />